config/default-ocp/manager_auth_proxy_patch.yaml

```diff
@@ -0,0 +1,22 @@
+# This patch inject a sidecar container which is a HTTP proxy for the
```

Comment:
I dropped this from upstream yesterday. Let's remove this. 

Kueue doesn't use kube-rbac-proxy from 0.11 onwards.